lib/hooks/useMarketImage.ts

```diff
@@ -0,0 +1,27 @@
+import { CATEGORY_IMAGES } from "lib/constants/category-images";
+import { useMarketCmsMetadata } from "./queries/cms/useMarketCmsMetadata";
+import { FullMarketFragment } from "@zeitgeistpm/indexer";
+
+export const useMarketImage = (
+  market: FullMarketFragment | { marketId: number; tags?: string[] },
+  opts?: {
+    fallback?: string;
+  },
+) => {
+  const firstTag = market.tags?.[0];
```

Comment:
I think this is an opportunity to add more variance to images. Instead of hardcoding to the first index we can choose an index with something like: `const tagIndex = market.marketId % tags.length`